platform/bitcoin/api.go

```diff
@@ -5,8 +5,8 @@ import (
 	"github.com/spf13/viper"
 	"github.com/trustwallet/blockatlas"
 	"github.com/trustwallet/blockatlas/coin"
+	"github.com/deckarep/golang-set"
```

Comment:
How is it library?

Reply:
A simple set type for the Go language. Also used by Docker, 1Password, Ethereum.